### PR TITLE
fix: bind keyItemSets to CloudServiceUsageOverviewDetailModal and refresh filters in CloudServiceDetailPage

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

This PR includes some refactoring and very simple bug fix code.
Please review in order by commits.

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/cloud-service/cloud-service-detail/CloudServiceDetailPage.vue [hunk 1]
```diff
@@ -268,15 +268,17 @@ export default {
         const cloudServiceDetailPageState = cloudServiceDetailPageStore.$state;
 
         const vm = getCurrentInstance()?.proxy as Vue;
-        const queryHelper = new QueryHelper();
 
         /* Main Table */
+        const queryTagsHelper = useQueryTags({});
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/cloud-service/cloud-service-detail/CloudServiceDetailPage.vue:273]: 🫶🏻

**Review by skdud4659 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-usage-overview/CloudServiceUsageOverviewDetailModal.vue [hunk 2]
```diff
@@ -229,7 +228,7 @@ export default defineComponent<Props>({
         /* Watchers */
         watch(
             [() => state.proxyVisible, () => props.schemaList, () => props.filters, () => props.hiddenFilters],
-            async ([visible, schemaList, filters, hiddenFilters], [, prevSchemaList, prevFilters, prevHiddenFilters]) => {
+            async ([visible, schemaList, filters, hiddenFilters], [, prevSchemaList]) => {
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-usage-overview/CloudServiceUsageOverviewDetailModal.vue:231]: praise: 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**